_includes/single-main-grid.html

```diff
@@ -0,0 +1,10 @@
+<div class="c-grid-item">
+<div class="c-grid-item__container">
+<a href="#" class="c-grid-item__link" aria-label="Visit First Officers page">
+    <img class="c-grid-item__image" src="assets/homepage-first-officer-2.png" alt="">
+    <div class="c-grid-item__button">
```

Comment:
Just looking at the code, I am a little confused about this class name and using the `div` tag. Is this going to have the styling of a button? And how does that would with the image next to it? I think if i saw it in the browser I would understand it better 🙂